twisted/test/test_paths.py

```diff
@@ -1196,8 +1196,9 @@ def test_createBinaryMode(self):
         self.assertIn("b", f.mode)
         f.write(b"\n")
         f.close()
-        read = open(path.path, "rb").read()
-        self.assertEqual(read, b"\n")
+        with open(path.path, "rb") as fp:
+            read = fp.read()
+            self.assertEqual(read, b"\n")
```

Comment:
`self.assertEqual(read, b"\n")` can go outside of the context manager.  I think it's clearer if it's dedented but I wouldn't hold back on merging this on account of that.